ops/model.py

```diff
@@ -1190,7 +1193,7 @@ def get_content(self, *, refresh: bool = False) -> Dict[str, str]:
         if refresh or self._content is None:
             self._content = self._backend.secret_get(
                 id=self.id, label=self.label, refresh=refresh)
-        return self._content
```

Comment:
Does it need to be a deepcopy?

I think this is significantly better than not doing it, I just wonder if secret contents are an object, can they have nested values?

Reply:
I've confirmed in the Juju `secret-get` and `secret-set` code that these are always plain `keystr=valstr` mappings, so there's never going to be nesting here.

In addition, for what it's worth, the Python type annotation is `Dict[str, str]`.